dataline-integrations/singer/stripe/source/entrypoint.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+set -e
+
+# TODO: replace with more general version
+# hacky version that expects ordering of args "--discover --config config.json"
+function check_connection() {
+  if [[ "$1" == "--config" ]]; then
```

Comment:
why assume position of params instead of iterating through and finding the right param? Unless this is going to go away very soon then it shouldn't be an issue.

Reply:
This will go away soon so I'm just going to leave it as is for now.